sdv/multi_table/hma.py

```diff
@@ -565,6 +565,9 @@ def _recreate_child_synthesizer(self, child_name, parent_name, parent_row):
                 )
                 synthesizer = self._synthesizer(table_meta, **self._table_parameters[child_name])
 
+            self._set_extended_columns_distributions(
+                synthesizer, child_name, self._parent_extended_columns[child_name]
```

Comment:
should the valid columns here be the child table's column instead of its parent's extended columns? These are supposed to be the columns that we want to skip when updating the distribution right?

Reply:
@amontanez24 `self._parent_extended_columns[child_name]` gets the list of extended columns on `child_name`, not its parent. These are the columns we want to use the default extended column distribution, instead of the table's default distribution.

Reply:
Ok, discussed offline, I had it backwards